### DISCUSSION TITLE
AlDente 1.26.1

### DISCUSSION THREAD

**Discussion by davidwernhart** [AppHouseKitchen/AlDente-Battery_Care_and_Monitoring#1208] [General]
### Pro

- Added the option to toggle the macOS Low Power Mode from AlDente's Feature settings.

- An indication of whether Lower Power Mode is active can now be found in the Sensor Data panel under "Power Specs".

- Added the option to choose what happens when you right-click on AlDente's menu bar icon.

- Fixed a bug that caused the screen to stay off when continuing the Calibration Mode in some scenarios.

- Fixed a bug that caused the charge limit to change after using the Calibration Mode.

- Removed the redundant "Hide Menubar Icon" setting. 

- Automatic Discharge will now start to engage 30 seconds after the app starts to increase compatibility with some Thunderbolt docks.

- Improved Power Flow accuracy for some devices.

- When Automatic Discharge is enabled and the MacBook is used in Clamshell Mode, AlDente now waits 15 seconds to start discharging after the cable is plugged in. This increases the compatability with some Thunderbolt 4 docks.

- Fixed a bug that caused charge limit issues when Sailing Mode was enabled and the battery was at 20%.

### All

- Added Serbian translation.

- Updated Polish and Russian translations.

- Fixed a bug that caused the app to crash when enabling sleep in some scenarios.

- Fixed a bug that caused MacBooks with Intel Mode enabled to always charge to 100% when powered off.

- Added a safety feature to prevent the charge limit from getting stuck on some Apple Silicon devices.

- AlDente now locally saves some sensor data values for visualization in future updates.

With peace & love from Vienna,
AppHouseKitchen

<hr /><em>This discussion was created from the release <a href='https://github.com/AppHouseKitchen/AlDente-Charge-Limiter/releases/tag/1.26.1'>AlDente 1.26.1</a>.</em>

**Comment by telepati** [comment]
Didn't you suggest that we select Low Power mode as Never? Is there no problem using this feature now?

**Reply by MatthiasKerbl** [reply]
Hi @telepati,

There was never an issue with the Low Power Mode and AlDente.
Is it possible that you mean the "Optimized Battery Charging" option?

Best,
Matthias

**Reply by telepati** [reply]
No, I'm mentioning Low Power mode. There is no option called "Optimized Battery Charging" in M3 Pro Sonoma anyway. Then there is no problem activating it.

**Reply by valynor** [reply]
@MatthiasKerbl 

Just clicked "check for updates" on my MBA 2020 with 1.25.2 several times, nothing happens. No download for 1.26.1 offered.  

Edit: restarted the app, now it finds the update. Maybe a bug in there somewhere. ;-)

**Reply by MatthiasKerbl** [reply]
Hi @valynor,

Thanks for letting us know.
For us the "Check for Updates" worked flawlessly without restarting AlDente.

Best,
Matthias

**Comment by Knesie210** [comment]
Hi @MatthiasKerbl, I was reading that there is the intel option which allows me to use alDente even if I'm in boot Camp, however I couldn't find it, I tried with old versions and the only way it appear was removing the helper, but then it didn't work, maybe can you help me? 
Thanks 

I use a mid 2015 monterey

**Reply by MatthiasKerbl** [reply]
Hi @Knesie210,

On your MacBook model, AlDente automatically works in Bootcamp.

Best,
Matthias
